docs/developer/prebuilt_world_loading.md

```diff
@@ -1,94 +1,126 @@
 
-    1. Download the Tomcat_Preloaded world from this google drive link: 
 
-https://drive.google.com/drive/folders/1sA5F7ifgzglJnxFaeNuXHFwdMjmrpNHB?usp=sharing
+**Getting Started:**
 
-	and save it somewhere convenient. Unzip the file. Henceforth Tomcat_Preloaded refers
-	to the folder that you got from unzipping.
-	**NOTE: Ensure the path to where you save this folder has no white spaces.
-	Ex:  Documents/folder name is not allowed but Documents/folder_name is allowed**
+1\. Download the SAR_v1 world from this Drive link: 
 
-    2. Download the file_gen and file_gen_skeleton XML s from this google drive link:
+    https://drive.google.com/open?id=19PjgHmUeJmOQ4DS4dAy-4bgax2_TnJRp
 
-https://drive.google.com/drive/folders/1sA5F7ifgzglJnxFaeNuXHFwdMjmrpNHB?usp=sharing
+Save it somewhere convenient and unzip the file. 
 
-	file_gen will serve as a reference for you to use file_gen_skeleton to get your pre-built world 	working. DO NOT SAVE IT INSIDE Tomcat_Preloaded
-	 I saved mine in: 
-	/home/adi-ua/Documents/TOMCAT/tomcat/external/malmo/sample_missions
+Henceforth, SAR_v1 refers to the folder you got from unzipping.
+	
+**NOTE** : Ensure the path to where you save this folder has no white spaces.
+
+Ex:  Documents/folder name is not allowed, but Documents/folder_name is allowed
+
+
+2\. Download the file_gen and file_gen_skeleton XMLs from this Drive link:
+
+    https://drive.google.com/drive/folders/1sA5F7ifgzglJnxFaeNuXHFwdMjmrpNHB?usp=sharing
+
+file_gen will serve as a reference for you to use file_gen_skeleton to get your pre-built world working.
+
+**DO NOT** save it inside  SAR_v1
+
+This is because the _entire_ SAR_v1 folder represents the Minecraft world. It is not meant to hold anything else.
+
+I saved mine in: 
+/home/$USER/Documents/TOMCAT/tomcat/external/malmo/sample_missions to keep it with all the other missions
+
+
+3\.	Open file_gen_skeleton.xml.
+
+Don’t change the name after we modify it (the next step) for the sake of this tutorial.
 
-3.	Open file_gen_skeleton.xml. Don’t change the name after we modify it for the sake of the 	tutorial
 
-4.	The comments in the XML indicate what needs to be filled in generally, but for the tutorial go to 	the line <FileWorldGenerator src = “” />
+4\.	Modifying the XML
 
-	Within the quotes specify the path to which you saved your Tomcat_Preloaded folder download.
-	The Minecraft world is the whole folder and not just one file inside the folder, so the path 	should end in Tomcat_Preloaded
+The comments in the XML indicate what needs to be filled in generally, but for
+the tutorial go to the line `<FileWorldGenerator src = “” />`
+
+Within the quotes specify the path to which you saved your SAR_v1 folder.
+
+Again,the **Minecraft world is the whole folder and not just one file inside the folder, so the path should end in:** SAR_v1
 	
-	Refer to file_gen.xml to see how I specified my path. 
+Refer to file_gen.xml to see how I specified my path. 
```

Comment:
Since you have references to yourself in the first-person singular, I think it would be good for you to put your name as the author of this tutorial. Perhaps you can add something like

```markdown
**Author:** Aditya Banerjee
**Date:** 2019-11-23
```

at the top (after the title)?